vertx-web-graphql/src/main/asciidoc/index.adoc

```diff
@@ -199,6 +199,54 @@ Here you can find how to configure the Apollo SubscriptionClient: https://github
 
 IMPORTANT: A subscription `DataFetcher` has to return a `org.reactivestreams.Publisher` instance.
 
+=== GraphQL multipart request
+
+To use https://github.com/jaydenseric/graphql-multipart-request-spec which will allow you to upload one or more files, using a single mutation call.
+
+To enable it, create the {@link io.vertx.ext.web.handler.graphql.GraphQLHandler} with options:
+
+[source,$lang]
+----
+GraphQLHandlerOptions graphQLHandlerOptions = new GraphQLHandlerOptions()
+  .setRequestMultipartEnabled(true);
+----
+
+Next, configure the router with the `BodyHandler`. This handler will parse the multipart HTTP body.
```

Comment:
When using Vert.x class names in the doc, put them in a Javadoc link reference. Then the doc will have a link to the Javadoc:

```suggestion
Next, configure the router with the {@link io.vertx.ext.web.handler.BodyHandler}. This handler will parse the multipart HTTP body.
```